app/Mail/LoginLinkMail.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Mail;
+
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class LoginLinkMail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    //Attribute
+    protected User $user;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Get the message envelope.
+     *
+     * @return \Illuminate\Mail\Mailables\Envelope
+     */
+    public function envelope()
+    {
+        return new Envelope(
+            subject: 'Dein persönlicher Login-Link',
```

Comment:
It would be good to name the "BIT" i think. so the recipient knows what login-link he received

Reply:
The genera is work in progress, because it is your work to create a proper email :)

Reply:
_Votekick_ @Chrissydrx